content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,147 @@
+title: Empower your documentation with addons
+date: 2024-04-18
+description: Read the Docs Addons let documentation authors add features to their documentation such as analytics, version feedback, and search as you type to documentation in a modular way where authors have control.
+category: Changelog
```

Comment:
I feel like this isn't a changelog entry. It's marketing or a feature announcement, I'd say.

Reply:
```suggestion
category: Feature announcement
```